js/views/userPage/UserPage.js

```diff
@@ -5,18 +5,43 @@ import app from '../../app';
 import Home from './UserPageHome';
 import Store from './UserPageStore';
 import Follow from './UserPageFollow';
+import Reputation from './UserPageReputation';
 
 export default class extends BaseVw {
   constructor(options = {}) {
     super(options);
     this.options = options;
 
+    this.tab = options.tab || 'Store';
     this.tabViewCache = {};
-    this.tabViews = { Home, Store, Follow };
+    this.tabViews = { Home, Store, Follow, Reputation };
 
-    this.followed = false; // TODO check to see if user is followed by the viewer
-    this.followsYou = true; // TODO check to see if this user follows the viewer
     this.ownPage = this.model.id === app.profile.id;
+
+    if (!this.ownPage) {
+      this.followed = app.ownFollowing.where({ guid: this.model.id }).length > 0;
```

Comment:
Since the userShort model has it's idAttribute set to the guid field, it's indexed by that field, so `app.ownFollowing.get(<guid>)' would be much more efficient than where which has to go through the entire list until it finds the user you're looking for.

Can you also make the switch to the other place you're using 'where' in this way.